Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py

```diff
@@ -114,3 +117,36 @@ def test_parse_resource_ids(test_input, expected_output):
     """
     from AWS_SecurityHub import parse_resource_ids
     assert parse_resource_ids(test_input) == expected_output
+
+
+class MockClient:
+
+    def get_findings(self, **kwargs):
+        return {'Findings': [
```

Comment:
better to set the findings object as a const so you wont have to initialize it here and in the test itself